src/utils/RouteConfig.js

```diff
@@ -58,7 +57,6 @@ export const routes = [
   { name: translate('services'), slug: '/services',
     items: [
       { name: translate('information'), slug: '/services/info', component: ServiceInfo },
-      { name: translate('packages.openstack'), slug: '/services/openstack-packages', component: OpenStackPackages },
       { name: translate('packages.ardana'), slug: '/services/openstack-ardana', component: ArdanaPackages },
```

Comment:
For consistency with the menu name, it would be better if this slug were renamed to /services/packages

Reply:
good suggestion! done!